enhancements/builds/entitlements.md

```diff
@@ -468,9 +383,23 @@ N/A
 
 ## Alternatives
 
-A build only solution could entail the build controller replacing the CSI plugin and admission framework and mounting
+1. A build only solution could entail the build controller replacing the CSI plugin and admission framework and mounting
 secrets into the build pod.
 
+2. CRI-O already has config options (see the [config file doc](https://github.com/cri-o/cri-o/blob/master/docs/crio.conf.5.md))  to specify host directories that should always be mounted into containers (i.e. `/etc/containers/mounts.conf` to override the default `/usr/share/containers/mounts.conf`), akin to what the Red Hat Docker Daemon used to provide with OpenShift V3.  Guidance in the early 4.x time frame has been around the cluster admin using the MCO post install to effectively update the filesystem of each node with the necessary subscription related files in a well known location that CRI-O would look for.  There is concern though using this through MCO from a couple of perspectives.  One, the MCO api is not natural for someone who wants to do something higher level like "enable entitlements for builds".  Second, using the MCO for any node update requires a restart of the node, which is onerous for this scenario.  Also note, the RHCOS team want to disable their current default setting of auto-mounting secrets from the host (i.e. creating an empty `/etc/containers/mounts.conf`), since RHCOS does not fully install subscription manager (and never will), and they have problems when only partial subscription manager metadata is available.  [RHCOS has the subscription-manager-certs installed but are missing the subscription-manager package which carries the rhsm.conf.](https://bugzilla.redhat.com/show_bug.cgi?id=1783393)
```

Comment:
```suggestion
2. CRI-O already has config options (see the [config file doc](https://github.com/cri-o/cri-o/blob/master/docs/crio.conf.5.md))  to specify host directories that should always be mounted into containers (i.e. `/etc/containers/mounts.conf` to override the default `/usr/share/containers/mounts.conf`), akin to what the Red Hat Docker Daemon used to provide with OpenShift V3.  Guidance in the early 4.x time frame has been around the cluster admin using the MCO post install to effectively update the filesystem of each node with the necessary subscription related files in a well known location that CRI-O would look for.  There are some concerns though about using MCO for this purpose.  One, the MCO api is not natural for someone who wants to do something higher level like "enable entitlements for builds".  Second, using the MCO for any node update requires a restart of the node, which is onerous for this scenario.  Also note, the RHCOS team wants to disable their current default setting of auto-mounting secrets from the host (i.e. creating an empty `/etc/containers/mounts.conf`), since RHCOS does not fully install subscription manager (and never will), and they have problems when only partial subscription manager metadata is available.  [RHCOS has the subscription-manager-certs installed but are missing the subscription-manager package which carries the rhsm.conf.](https://bugzilla.redhat.com/show_bug.cgi?id=1783393)
```